server/src/main/java/org/opensearch/identity/tokens/StandardTokenClaims.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.tokens;
+
+/**
+ * This enum represents the standard claims that are used in the JWTs for OnBehalfOf tokens
+ */
+public enum StandardTokenClaims {
```

Comment:
This is pulling JWT concepts into OpenSearch, I'm not sure it needs them, I'd rather have these abstracted and what OpenSearch uses makes sense for its scenarios, not for the external system.

Reply:
I probably won't get a chance to address this this week, but this is what Craig asked for. I thought Ryan was taking this over this week, but let's just go with a more generic approach and make things specific later. If this has not been addressed by next week I am going to pull this back out and re-generify the concepts from before where appropriate.

Reply:
@scrawfor99 I will create a PR against your fork like the `toString()` one I did.

Reply:
Removed